Automattic/wp-calypso
# Stats: Move module-tabs

As part of the stats maintenance project, this branch moves `module-tabs` and `module-tab` into their own sub component directory in `stats`.  No .scss has been moved with this branch, but the components were refactored with es6 syntax.

__To Test__
- Load a site stats page
- Click on any of the date period filters ( days, weeks etc )
- Note that the tabs below the chart look the same
- Click on the chart tabs, ensure the chart toggles as expected

/cc @alternatekev
